code/model/DMSDocument.php

```diff
@@ -1249,6 +1257,16 @@ public function init()
     }
 
     /**
+     * Allow files to be opened in a browser as the default value. This can be superceded by
+     * values set on a per-document basis.
+     * @static
+     * @param $behavior string (open,download)
+     */
+    public static function default_download_behavior( $behavior ){
```

Comment:
I'm not quite sure what you mean here. Do you mean to set default config values in YML and then facilitate overwriting via `/mysite/_config/mysite.yml`?